src/schemathesis/specs/openapi/links.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Dict, List
+
+import attr
+
+from ...models import Endpoint
+from ...utils import GenericResponse
+
+
+@attr.s(slots=True)
+class Link:
+    name: str = attr.ib()
+    endpoint: Endpoint = attr.ib()
+    parameters: Dict[str, Any] = attr.ib()
+
+    @classmethod
+    def from_definition(cls, name: str, definition: Dict[str, Dict[str, Any]], source_endpoint: Endpoint) -> "Link":
+        endpoint = None
+        for endpoint in source_endpoint.schema.get_all_endpoints():
+            # it could be better to have O(1) access via a mapping by operationId or operationRef
+            if endpoint.definition.raw.get("operationId") == definition.get("operationId"):
```

Comment:
I think that it will be much better to build a structure to access by a key instead of iterating each time